velox/type/DecimalUtil.h

```diff
@@ -304,6 +304,22 @@ class DecimalUtil {
     }
   }
 
+  inline static std::optional<int128_t> computeValidSum(
```

Comment:
Maybe add some comments and corresponding UT for it.

Reply:
> Maybe add some comments and corresponding UT for it.

Done.